src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -966,3 +966,124 @@
+    text: >
+        az acr scope-map update -n MyScopeMap -r MyRegistry --remove hello-world metadata/read content/read --remove hello-world-again metadata/write
+"""
+
+helps['acr scope-map show'] = """
+type: command
+short-summary: Show details and attributes for a scope map under an Azure Container Registry.
+examples:
+  - name: Get information for the scope map 'MyScopeMap'.
+    text: >
+        az acr scope-map show -n MyScopeMap -r MyRegistry
+"""
+
+helps['acr scope-map list'] = """
+type: command
+short-summary: List all scope maps under an Azure Container Registry.
+examples:
+  - name: List scope maps under the registry 'MyRegistry'.
+    text: >
+        az acr scope-map list -r MyRegistry
+"""
+
+helps['acr token'] = """
+type: group
+short-summary: Manage tokens for an Azure Container Registry.
+"""
+
+helps['acr token create'] = """
+type: command
+short-summary: Create a token for an Azure Container Registry.
+examples:
+  - name: Create a token associated to the scope map 'MyScopeMap', in 'disabled' status.
+    text: >
+        az acr token create -n MyToken -r MyRegistry --scope-map MyScopeMap --status disabled
+"""
+
+helps['acr token delete'] = """
+type: command
+short-summary: Delete a token under an Azure Container Registry.
+examples:
+  - name: Delete the token 'MyToken'.
+    text: >
+        az acr token delete -n MyToken -r MyRegistry
+"""
+
+helps['acr token update'] = """
+type: command
+short-summary: Update a token under an Azure Container Registry, replacing the scope map associated to it.
+examples:
+  - name: Update the token 'MyToken', making it associated with the scope map 'MyNewScopeMap'.
+    text: >
+        az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
+"""
+
+helps['acr token show'] = """
+type: command
+short-summary: Show details and attributes for a token under an Azure Container Registry.
+examples:
+  - name: Get information for the token 'MyToken'.
+    text: >
+        az acr token show -n MyToken -r MyRegistry
+"""
+
+helps['acr token list'] = """
+type: command
+short-summary: List all tokens under an Azure Container Registry.
+examples:
+  - name: List tokens under the registry 'MyRegistry'.
+    text: >
+        az acr token list -r MyRegistry
+"""
+
+helps['acr token credential'] = """
+type: group
+short-summary: Manage credentials entities for a token under an Azure Container Registry.
+"""
+
+helps['acr token credential generate'] = """
+type: command
+short-summary: Generate both passwords for a token under an Azure Container Registry, unless one password is specified. Replaces old values if they already exists. If expiry is not specified, it will be defaulted to the max value "9999-12-31T23:59:59.9999+00:00".
+examples:
+  - name: Generate password1 for the token 'MyToken', with expiration for 2 years.
+    text: az acr token credential generate -n MyToken -r MyRegistry --password1 --months 24
```

Comment:
If the creds are meant to be used by `docker login` or `az acr login`, I would suggest we call out so that users would be able to have end to end connected.

Reply:
Suggested adding a link to our docs from the summary section.

Reply:
The suggested link aka.ms/acr/token has been added now.